Resources/Prototypes/Entities/Structures/Power/Generation/portable_generator.yml

```diff
@@ -327,22 +323,20 @@
     - type: Machine
       board: PortableGeneratorJrPacmanMachineCircuitboard
     - type: FuelGenerator
-      targetPower: 6000 # Frontier - 2000<6000
-      minTargetPower: 1000
-      optimalPower: 6000 # Frontier - 2000<6000
+      targetPower: 10000
+      minTargetPower: 4000
+      optimalPower: 10000
       maxTargetPower: 15000
-      # 30~ minutes at full tank.
-      optimalBurnRate: 0.11111111
-      # Shallow curve
-      fuelEfficiencyConstant: 0.5
+      clockstepConstant: 8000
     - type: ChemicalFuelGeneratorAdapter
       solution: tank
       reagents:
         WeldingFuel: 1
+        Oil: 5
```

Comment:
Great PR! I like the changes to power systems =3

Whats this change do? Does this make jr pacmans run on oil, or like, did they always run on oil?